src/gc.c

```diff
@@ -392,12 +392,36 @@ static void run_finalizers(jl_ptls_t ptls)
     arraylist_free(&copied_list);
 }
 
+JL_DLLEXPORT int jl_gc_get_enable_finalizers(jl_ptls_t ptls)
```

Comment:
```suggestion
JL_DLLEXPORT int jl_gc_get_finalizers_inhibited(jl_ptls_t ptls)
```

Reply:
I named it after the method name, rather than the field. But it's exposed as a test-only API currently, partly because it seemed like it shouldn't have a real use and need exporting, but also because I didn't want to name it

Reply:
Yes, I see it's test-only, but it's still confusing for it to return the opposite of what the name seems to say.